src/toolcalls/erc20Conversion.ts

```diff
@@ -74,7 +72,7 @@ export class ERC20ConversionMessage
   ];
 
   inProgressComponent() {
-    return this.hasValidWallet ? InProgressTxDisplay : ConnectWalletPrompt;
```

Comment:
We are temporarily losing a bit of functionality here, but we have the opportunity to refactor  the nicely designed `InProgressTxDisplay` to display "Connect your wallet to continue' if no valid wallet is present.